src/Transports.AspNetCore/Transports.AspNetCore.csproj

```diff
@@ -10,10 +10,17 @@
 
   <ItemGroup>
     <PackageReference Include="GraphQL" Version="$(GraphQLVersion)" />
-    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="$(MicrosoftAspNetCoreHostingAbstractionsVersion)" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="$(MicrosoftExtensionsOptionsVersion)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsNetCore30OnwardsTarget)' == 'True'">
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
The same question.

Reply:
Same answer - we need stuff that was previously in `Microsoft.AspNetCore.Hosting.Abstractions` and is now only available in the shared framework.